src/cli/run.ts

```diff
@@ -26,7 +26,7 @@ export async function run(options: RuleOptions = {}) {
 
   const pathFlatConfig = path.join(cwd, 'eslint.config.js')
   const pathPackageJSON = path.join(cwd, 'package.json')
-  const pathESLintIngore = path.join(cwd, '.eslintignore')
+  const pathESLintIgnore = path.join(cwd, '.eslintignore')
```

Comment:
You might also need to update the references

Reply:
Sorry, I just edited it on github, thanks for your tip!